build.sbt

```diff
@@ -8,20 +8,20 @@ val Org = "org.scoverage"
 val ScalatestVersion = "3.1.1"
 
 val bin212 = Seq("2.12.13", "2.12.12", "2.12.11", "2.12.10")
-val bin213 = Seq("2.13.4", "2.13.3", "2.13.2", "2.13.1", "2.13.0")
+val bin213 = Seq("2.13.5", "2.13.4", "2.13.3", "2.13.2", "2.13.1", "2.13.0")
```

Comment:
Also if you are adding in 2.13.5 right away, can you also ensure the tests run for that version.

Reply:
I hit on some issues locally when adding 2.13.5 btw.

```
Error downloading org.scala-js:scalajs-compiler_2.13.5:1.0.0
```

I think it's due to the version of the scalajs compiler that we are using. It might work to bump that up to 1.0.5

Reply:
Thanks. I Added them.

Reply:
As for the scalajs-compiler. Yeah it is only published for 2.13.5 with verison 1.5.0 onwards